src/utils/fetch-bounties.ts

```diff
@@ -32,6 +33,20 @@ export async function fetchBountiesAndChildBounties(showProgress = true) {
 			childBounties.sort((a, b) => (a.id > b.id ? -1 : 1));
 		});
 
+		await fetchAllProxies();
```

Comment:
The function `addBountiesFromDoTreasury` above also does a similar job: enriching the bounties data from an external source. For consistency, would it be helpful to move the multisig enrichment into a separate function?

Reply:
will do in a seperate PR